report/src/main/webapp/app/layout/main/url/caseFactory.service.js

```diff
@@ -162,14 +162,24 @@ define(['angularAMD', 'artifactsService'], function (angularAMD) {
     }
 
     function initializeCollectorResultArtifact() {
-      var collectorHasResult = caseModel.step.stepResult && caseModel.step.stepResult.artifactId;
+      var collectorHasResult = caseModel.step.stepResult && caseModel.step.stepResult.artifactId,
+          sourceCollector = caseModel.step.type === 'source',
+          artifactId;
+
       if (collectorHasResult) {
-        artifactsService.getArtifact(caseModel.step.stepResult.artifactId)
-            .then(function (data) {
-              caseModel.collectorResult = data;
-            }).catch(function (e) {
-          console.log(e);
-        });
+        artifactId = caseModel.step.stepResult.artifactId;
+        if (sourceCollector) {
+          caseModel.collectorResult = artifactsService.getArtifactUrl(artifactId);
+        } else {
+          console.log('Getting: ', artifactId);
```

Comment:
Does it really work? Shouldn't it be `log('Getting: ' + artifactId)` or `log('Getting: %s', artifactId)`?